CHANGELOG-1.18.md

```diff
@@ -85,7 +85,7 @@ filename | sha512 hash
 
 ### Feature
 
-- aggragation api will have alpha support for network proxy ([#87515](https://github.com/kubernetes/kubernetes/pull/87515), [@Sh4d1](https://github.com/Sh4d1)) [SIG API Machinery]
+- aggregation api will have alpha support for network proxy ([#87515](https://github.com/kubernetes/kubernetes/pull/87515), [@Sh4d1](https://github.com/Sh4d1)) [SIG API Machinery]
```

Comment:
i believe these entries are generated at this point.
also the release note tool had a mechanic for fixing typos on a search and replace principle.

alternatively you can try asking the PR author to fix the typo in #87515
/cc @Sh4d1

/sig release
/kind documentation

Reply:
@neolit123 PR updated :)